### PR TITLE
Update ISSUE__TEMPLATES to point to Redot-Engine repo instead of Godot repo

### PR DESCRIPTION
**Pull request by SympleSynz**
Only updating the config.yml and bug_report.yml to point to the Redot-Engine repos instead of the Godot repos

### CODE REVIEW

**Review by filipworksdev — APPROVED** [review]

**Review by filipworksdev — APPROVED** [review]

**Comment by SympleSynz** [comment]
Can't be merged in until it is approved by a maintainer.

**Comment by filipworksdev** [comment]
Merged. Thanks! 
The documentation links are broken. Should still point at Godot's documentation but is okay. We can fix later.

**Pull request merged**
